pkg/commons/config.go

```diff
@@ -30,3 +30,15 @@ type PineconeConfig struct {
 	Host      string `required:"true"  help:"Pinecone index host URL (e.g., https://example-index-12345.svc.region.pinecone.io)"`
 	Namespace string `help:"Namespace of the partition to migrate"`
 }
+
+type ChromaConfig struct {
+	Collection  *string `required:"true" help:"Chroma collection name"`
+	Url         *string `help:"Chroma server URL" default:"http://localhost:8000"`
```

Comment:
Pointer types let us differentiate between zero values and no value being passed from the command line.